.github/workflows/update-help-command.yml

```diff
@@ -1,13 +1,14 @@
 # Name of this GitHub Actions workflow.
-name: Update help command
+name: Update semgrep scan help command
```

Comment:
Certainly this is not a blocker, but would you like to rename the file to match this new name as well?